tsdate/prior.py

```diff
@@ -970,10 +973,22 @@ def fill_priors(node_parameters, timepoints, ts, Ne, *, prior_distr, progress=Fa
     datable_nodes = np.ones(ts.num_nodes, dtype=bool)
     datable_nodes[ts.samples()] = False
     datable_nodes = np.where(datable_nodes)[0]
+
+    """
```

Comment:
Can we put these as comments (starting with #) rather than triple quoted strings, we we tend to reserve only for docstrings at the start of the function